ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-release-pipeline.yml

```diff
@@ -0,0 +1,178 @@
+      params:
+        - name: bundle_path
+          value: "$(params.bundle_path)"
+      workspaces:
+        - name: source
+          workspace: repository
+          subPath: src
+
+    # Get the bundle versions to build index
+    - name: get-supported-versions
+      runAfter:
+        - checkout
+      taskRef:
+        name: get-supported-versions
+      params:
+        - name: bundle_path
+          value: $(params.bundle_path)
+      workspaces:
+        - name: source
+          workspace: repository
+          subPath: src
+
+    # This task is needed only because of it's second step, which
+    # parses the bundle path and returns the package_name and bundle_version.
+    # TODO: split this task
+    - name: operator-validation
+      runAfter:
+        - checkout
+      taskRef:
+        name: operator-validation
+      params:
+        - name: bundle_path
+          value: "$(params.bundle_path)"
+        - name: pyxis_url
+          value: "$(params.pyxis_url)"
+      workspaces:
+        - name: source
+          workspace: repository
+          subPath: src
+
+    # Get additional data necessary for certification
+    - name: get-bundle-related-data
+      runAfter:
+        - certification-project-check
+      taskRef:
+        name: get-bundle-related-data
+      params:
+        - name: cert_project_id
+          value: "$(tasks.certification-project-check.results.certification_project_id)"
+
+    # Publish image to OCP registry
+    - name: imagestream-manipulation
+      runAfter:
+        - get-bundle-related-data
+        - operator-validation
+      taskRef:
+        name: imagestream-manipulation
+      params:
+        - name: cert_project_id
+          value: "$(tasks.certification-project-check.results.certification_project_id)"
+        - name: vendor_label
+          value: "$(tasks.get-bundle-related-data.results.vendor_label)"
+        - name: repository_name
+          value: "$(tasks.get-bundle-related-data.results.repository_name)"
+        - name: isv_pid
+          value: "$(tasks.get-bundle-related-data.results.isv_pid)"
+        - name: imagestream
+          value: $(params.imagestream)
+        - name: tag
+          value: $(params.tag)
+        - name: dist_method
+          value: $(params.dist_method)
+        # params necessary to pull the image
+        - name: registry
+          value: "$(params.registry)"
+        - name: image_namespace
+          value: "$(params.image_namespace)"
+        - name: package_name
+          value: "$(tasks.operator-validation.results.package_name)"
+        - name: bundle_version
+          value: "$(tasks.operator-validation.results.bundle_version)"
+
+    # Publish Vendor, Repository
+    - name: publish-resources
+      runAfter:
+        - imagestream-manipulation
+      taskRef:
+        name: publish-resources
+      params:
+        - name: cert_project_id
+          value: "$(tasks.certification-project-check.results.certification_project_id)"
```

Comment:
We should probably publish the vendor and repo after the image is published. Given this order, if the repo gets published and then the pipeline fails, the repo will appear on the catalog search but the detail page may fail to load because of the lack of published image metadata.